src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java

```diff
@@ -2,69 +2,72 @@
 
 import com.softserveinc.ita.dkrutenko.pageobjects.softserve.ContactUsPage;
 import com.softserveinc.ita.dkrutenko.pageobjects.softserve.MainPage;
-import com.softserveinc.ita.dkrutenko.utils.runners.TestRunner;
+import com.softserveinc.ita.dkrutenko.pageobjects.softserve.models.User;
+import com.softserveinc.ita.dkrutenko.pageobjects.softserve.models.repo.UserRepo;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static com.codeborne.selenide.Selenide.open;
 
-public class SoftServeTests extends TestRunner {
-
-    private final MainPage softServeBasePage = new MainPage();
+public class SoftServeTests {
+    protected static final String softServeUrl = "https://www.softserveinc.com/";
```

Comment:
if it's used only in before method, move it directly there

Reply:
done